### ISSUE TITLE
Making entry points in python

### ISSUE THREAD

**Issue by arita37** [mkaz/termgraph#110]
Very good job 
Thanks !

Is it possible to have simple python entrypoints

from ... import histogram, scatter

so we can use from python too !

**Comment by mkaz** [comment]
Yes, it exists. See [module-test1.py](https://github.com/mkaz/termgraph/blob/main/tests/module-test1.py) and [module-test2.py](https://github.com/mkaz/termgraph/blob/main/tests/module-test2.py) for examples.

With termgraph installed, run with python:

```
python -m pip install termgraph
python module-test1.py
```
